### DISCUSSION TITLE
Apache NetBeans 22 release candidates

### DISCUSSION THREAD

**Discussion by ebarboni** [apache/netbeans#7281] [General]
Release candidates for the next release of Apache NetBeans will be available weekly until release. Your help in testing, reporting and verifying issues would be highly appreciated by the whole Apache NetBeans community.

Changes from NetBeans 21 can be seen at https://github.com/apache/netbeans/releases/tag/22-rc1

## Binary / source zip

Binary and source zips are available from -

https://nightlies.apache.org/netbeans/candidate/netbeans/

## Installers

Unsigned Windows, Linux DEB and Linux RPM installers are available from -

https://nightlies.apache.org/netbeans/candidate/installers/

NB. We are working on making other installers available for release candidates from the same location.

## VSCode plugin

The VSCode plugin is available from - 

https://nightlies.apache.org/netbeans/candidate/vsix/

## IMPORTANT

These release candidates are pre-release software, and not official releases of Apache NetBeans.

**Comment by neilcsmith-net** [comment]
A signed and notarized Apple installer for 22-rc3 has been added to https://nightlies.apache.org/netbeans/candidate/installers/  This installs alongside the main release package for testing, and the app is called `ASF NetBeans RC` to differentiate.

If you're a macOS user, please help with testing this.

**Comment by kafumanto** [comment]
Hello, the URL <https://nightlies.apache.org/netbeans/candidate/netbeans/> returns a 404.

**Reply by ebarboni** [reply]
the vote is in progress, so candidate are no more available.The vote thread is on dev mailing list.

**Reply by kafumanto** [reply]
Ok, thanks!

**Reply by mbien** [reply]
going to unpin this discussion thread

**Comment by mbien** [comment]
closing, since NetBeans 22 is now released -> https://netbeans.apache.org/download/
